packages/webapp/pages/welcome/index.tsx

```diff
@@ -124,7 +143,7 @@ const DemoPage = (): ReactElement => {
           what&apos;s out there.
         </p>
       </div>
-      <Feed className={feedProps.className} {...feedProps} />
+      {featuresReady && <Feed className={feedProps.className} {...feedProps} />}
```

Comment:
Does this delay rendering? Because placeholders only happen inside here

Reply:
A bit, but not that much.

Reply:
Placeholder shows delayed either way since it is not in the first pain in SSR, but did not want to look at why the useFeed does that in this PR.

Reply:
`useFeed` placeholder is delayed due to `tokenRefreshed` condition on `enabled` but I don't want to remove it.